tests/test_custom_learner.py

```diff
@@ -229,3 +229,23 @@ def test_custom_learner_api_missing_file():
 def test_custom_learner_api_bad_extension():
     other_dir = join(_my_dir, 'other')
     _ = Learner('_CustomLogisticRegressionWrapper', custom_learner_path=join(other_dir, 'custom_learner.txt'))
+
+
+@raises(ValueError)
+def test_custom_learner_learning_curve_min_examples():
+    """
+    test to check that Learner.learning_curve raises an error
+    when the number of training examples provided is less than 500
```

Comment:
```suggestion
    Test to check learning curve raises error with less than 500 examples
```

We don't want these to be too long since they are hard to read when run in verbose mode.